src/components/CategoryMenu.jsx

```diff
@@ -1,18 +1,17 @@
 const CategoryMenu = () => {
   return (
-    <>
-      <label htmlFor="games category filter">Go to:</label>
-      <select>
-        <option disabled defaultValue={""} hidden>
-          Categories
-        </option>
-        <option></option>
-
-        <option>to</option>
-        <option>be</option>
-        <option>completed</option>
-      </select>
-    </>
+    <nav>
+      <label htmlFor="games category filter">Pick A Category:</label>
+      <ol className="category-nav">
+        <li className="category-option">Strategy</li>
+        <li className="category-option">Hidden-roles</li>
+        <li className="category-option">Dexterity</li>
+        <li className="category-option">Push-your-luck</li>
+        <li className="category-option">Roll-and-write</li>
+        <li className="category-option">Deck-building</li>
+        <li className="category-option">Engine-building</li>
```

Comment:
Think this bit is a future ticket, but could you use your api request to get the categories from the db?